### PR TITLE
Add CreatedAt field to the GetNodes response

### PR DESCRIPTION
**Pull request by lancewf**
image

### :nut_and_bolt: Description: What code changed, and why?

Adding the CreatedAt field to the GetNodes response

### :athletic_shoe: How to Build and Test the Change
1. `build components/config-mgmt-service && start_all_services`
1. Add some nodes with `chef_load_nodes 10`
1. Go to https://a2-dev.test/infrastructure/client-runs and check the response for the "https://a2-dev.test/api/v0/cfgmgmt/nodes" request. It should look like the above image. 

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated?
- [ ] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: components/config-mgmt-service/integration_test/nodes_test.go [hunk 1]
```diff
@@ -2663,115 +2663,115 @@ func TestNodesWithTableDriven(t *testing.T) {
...
-				Filter: []string{"status:success"},
-				Sorting: &request.Sorting{
-					Field: "name",
-					Order: request.Order_desc,
-				},
-				Pagination: &request.Pagination{Size: int32(index)},
-			},
-			expectedNodeListFromNodeArray(reverseNodesArray(filterNodesArray(nodes, "status:success")))},
+		// {"should return all nodes",
+		// 	request.Nodes{Pagination: &request.Pagination{Size: int32(index)}},
```
Comment (vjeffrey) [inline components/config-mgmt-service/integration_test/nodes_test.go:2667]: ?

Comment (lancewf) [reply components/config-mgmt-service/integration_test/nodes_test.go:2667]: oops! I will update.

File: components/config-mgmt-service/backend/client.go [hunk 2]
```diff
@@ -77,7 +77,7 @@ type Client interface {
 // service since it is the origin of our data. Therefore we will just import
 // them as our own so we can call them as `backend.Type`
 type Run ingest.Run
-type Node ingest.Node
+type Node ingest.UpsertNode
```
Comment (lancewf) [inline components/config-mgmt-service/backend/client.go:80]: The `UpsertNode` includes the create date field.

File: components/config-mgmt-service/grpcserver/nodes.go [hunk 3]
```diff
@@ -332,6 +332,15 @@ func backendNodeArrayToMessageArray(nodes []backend.Node,
...
 
+			createdProto, err := ptypes.TimestampProto(node.Created.UTC())
+			if err != nil {
+				log.WithFields(log.Fields{
+					"func":    nameOfFunc(),
+					"err":     err,
+					"lastCCR": node.Created.UTC().String(),
+				}).Warn("Unable to translate created time to timestamp proto")
+			}
+
```
Comment (lancewf) [inline components/config-mgmt-service/grpcserver/nodes.go:343]: If there is no `Created` field the date will be Jan 1st year 0001

**Pull request merged**